cmd/kops/integration_test.go

```diff
@@ -347,18 +341,6 @@ func TestDockerCustom(t *testing.T) {
 	newIntegrationTest("docker.example.com", "docker-custom").runTestCloudformation(t)
 }
 
-// TestLaunchConfigurationASG tests ASGs using launch configurations instead of launch templates
-func TestLaunchConfigurationASG(t *testing.T) {
-	featureflag.ParseFlags("-EnableLaunchTemplates")
-	unsetFeatureFlags := func() {
-		featureflag.ParseFlags("+EnableLaunchTemplates")
-	}
-	defer unsetFeatureFlags()
-
-	newIntegrationTest("launchtemplates.example.com", "launch_templates").withZones(3).withLaunchConfiguration().runTestTerraformAWS(t)
-	newIntegrationTest("launchtemplates.example.com", "launch_templates").withZones(3).withLaunchConfiguration().runTestCloudformation(t)
```

Comment:
With this being removed, we can remove everything under `tests/integration/update_cluster/launch_templates`